tests/fast_int.cpp

```diff
@@ -0,0 +1,379 @@
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+#include <iomanip>
+#include <cstring>
+#include "fast_float/fast_float.h"
+
+/*
+all tests conducted are to check fast_float::from_chars functionality with int and unsigned
+test cases include:
+int basic tests - numbers only, numbers with strings behind, decimals, negative numbers
+unsigned basic tests - numbers only, numbers with strings behind, decimals
+int invalid tests - strings only, strings with numbers behind, space in front of number, plus sign in front of number
+unsigned invalid tests - strings only, strings with numbers behind, space in front of number, plus/minus sign in front of number
+int out of range tests - numbers exceeding int bit size (Note: out of range errors for 8, 16, 32, and 64 bits have not been tested)
```

Comment:
out_of_range was the nastiest to get right and I'm not 100% sure my code is correct.
can you add test cases for every base for 64-bit (signed and unsigned)?